### PR TITLE
Move rumble files to CMSMonit from CMSKubernetes

### PR DESCRIPTION
**Pull request by mrceyhun**
@vkuznet 
Rumble related non-k8s related files moved to CMSMonitoring.

### CODE REVIEW

**Comment by mrceyhun** [comment]
I used tab in python code, pep8 saw it okay. According to your suggestion, I changed it to 4 spaces now.

File: src/go/Rumble/rumble.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	if verbose > 0 {
+		log.Println("Response Headers:", resp.Header)
+	}
+	if resp.StatusCode == http.StatusOK {
+		write(resp, output)
+	}
+}
+
+func main() {
+	defaultUrl := "http://test-cluster-jnbxujghdusq-node-0.cern.ch"
```
Comment (vkuznet) [inline src/go/Rumble/rumble.go:101]: please use https://cms-monitoring.cern.ch as default url in the code, in your tests you can use whatever url you want.

File: src/python/Rumble/rumble.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+# third party libs
+import requests
+
+
+class OptionParser:
+    def __init__(self):
+        """ User based option parser """
+        # Max output size is unlimited
+        default_url = "http://test-cluster-jnbxujghdusq-node-0.cern.ch"
```
Comment (vkuznet) [inline src/python/Rumble/rumble.py:20]: please use https://cms-monitoring.cern.ch as default url

**Review by vkuznet — CHANGES_REQUESTED** [review]
Please fix default urls, everything is ok.

**Comment by mrceyhun** [comment]
@vkuznet 
I rebased upstream after last merge and applied your requested changes.

**Pull request merged**